wordplay.py

```diff
@@ -0,0 +1,244 @@
+#Setting up all the words in a list 
+with open('sowpods.txt', 'r') as scrabble_words:
+    words = []
+    for word in scrabble_words:
+        words.append(word.strip())
+
+
+#Question: 
+#What are all of the words containing UU?
+words_with_UU = []
+for word in words:
+    if 'UU' in word:
+        words_with_UU.append(word)
+#to display answer: 
+#print(words_with_UU)
+
+
+#Question: 
+#What are all of the words containing an X and a Y and a Z
+words_with_X_Y_Z = []
+for word in words:
+    if "X" in word and "Y" in word and "Z" in word:
+       words_with_X_Y_Z.append(word) 
+#to display answer: 
+#print(words_with_X_Y_Z)
+
+#Question: 
+#What are all the words containing a Q but not a U
+words_with_Q_not_U = []
+for word in words:
+    if "Q" in word and "U" not in word:
+            words_with_Q_not_U.append(word)
+#To Display Answer:
+#print(words_with_Q_not_U)
+
+#Question: 
+#What are all of the words that contain the word CAT and are exactly 5 letters long?
+words_with_CAT_5 = [] 
+for word in words:
+    if len(word) == 5 and "CAT" in word:
+        words_with_CAT_5.append(word)
+#To Display Answer:
+#print(words_with_CAT_5)
+
+#Question: 
+#What are all of the words that have no E or A and are at least 15 letters long?
+words_no_E_A_15plus =[] 
+for word in words:
+    if len(word) >= 15:
+        if 'E' not in word and "A" not in word:
+            words_no_E_A_15plus.append(word)
+#To Display Answer:
+#print(words_no_E_A_15plus)
+        
+#Question: 
+#What are all of the words that have a B and an X and are less than 5 letters long?
+words_B_X_5less = []
+for word in words:
+    if len(word) < 5 and "B" in word and "X" in word:
+        words_B_X_5less.append(word)
+#To Display Answer:
+#print(words_B_X_5less)
+
+#Question: 
+#What are all of the words that both start and end with Y?
+words_startY_endY = []
+for word in words: 
+    if word[0] == "Y" and word[-1] == "Y":
+        words_startY_endY.append(word)
+#To Display Answer:
+#print(words_startY_endY)
+
+#Question: 
+#What are all of the words with no vowel and not even a Y?
+vowelsAndY = "AEIOUY"
+
+words_no_vowels_includingY = []
+for word in words:
+    vowel_in_word = False
+    for char in vowelsAndY:
+        if char in word:
+            vowel_in_word = True
+            break 
+    if not vowel_in_word:
+        words_no_vowels_includingY.append(word)
+# To display answer as an array
+#print(vowel_in_word)
+
+#Question: 
+# What are all of the words that have all 5 vowels, in any order?
+vowels = "AEIOU"
+words_with_all_vowels = [] 
+for word in words: 
+    vowel_in_word = True
```

Comment:
nit: naming

`vowel_in_word` may be a little misleading, something like `word_contains_all_vowels` may be a little more clear